content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
 ---
-title: "Fundamental Wellbeing"
-created: 2023-04-21
-authors: 
+title: An Introduction to Dr. Jeffery Martin's Work on Fundamental Wellbeing
```

Comment:
I think you want want to quote the title if you have `'` in it.

Reply:
I also think our original title on youtube was pretty good:

> Dr. Jeffrey Martin on A Scientific Approach to Awakening and Fundamental Wellbeing

Could tweak this:

A Scientific Approach to Awakening and Fundamental Wellbeing: the work of Dr. Jeffrey Martin

Reply:
I used your tweak